Application.groovy

```diff
@@ -1,7 +1,6 @@
-@Grab("edu.wisc.my.restproxy:json-proxy-service:1.0")
+@Grab("edu.wisc.my.restproxy:json-proxy-service:1.1.6-SNAPSHOT")
```

Comment:
Yeah, I only see the snapshot for 1.1, so I think a little config might be needed in the pom.
https://oss.sonatype.org/content/repositories/snapshots/edu/wisc/my/restproxy/json-proxy-service/